PersistenceMigrationsSample/app/src/androidTestRoom_Common/java/com/example/android/persistence/migrations/SqliteDatabaseTestHelper.java

```diff
@@ -23,7 +23,8 @@ public static void insertUser(int userid, String userName, SqliteTestDbOpenHelpe
     public static void createTable(SqliteTestDbOpenHelper helper) {
         SQLiteDatabase db = helper.getWritableDatabase();
 
-        db.execSQL("CREATE TABLE IF NOT EXISTS users (userid INTEGER PRIMARY KEY, username TEXT )");
+        db.execSQL("CREATE TABLE IF NOT EXISTS users (userid INTEGER PRIMARY KEY NOT NULL,"
```

Comment:
Change the [`UsersDbHelper`](https://github.com/googlesamples/android-architecture-components/blob/master/PersistenceMigrationsSample/app/src/sqlite/java/com/example/android/persistence/migrations/UsersDbHelper.java) also. Otherwise the test doesn't match the initial state of the app, using the sqlite database, without room.

Reply:
hmm, how did the test pass?